wgpu-hal/src/gles/queue.rs

```diff
@@ -560,7 +655,15 @@ impl super::Queue {
                 }
             }
             C::ClearColorF(draw_buffer, ref color) => {
-                gl.clear_buffer_f32_slice(glow::COLOR, draw_buffer, color);
+                if self
+                    .shared
+                    .private_caps
+                    .contains(super::PrivateCapabilities::REQUIRES_SHADER_CLEAR)
```

Comment:
This check is too general, we should only be doing this if we are clearing to an SRGB buffer.

Reply:
How would I know whether or not the buffer is sRGB?